common/ledger/util/leveldbhelper/pkg_test.go

```diff
@@ -49,12 +49,14 @@ func (dbEnv *testDBEnv) cleanup() {
 	if dbEnv.db != nil {
 		dbEnv.db.Close()
 	}
+	os.RemoveAll(dbEnv.path)
```

Comment:
this duplicates what we have in `assert.NoError`

Reply:
fixed.